CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# Contributing to CICE
+
+Welcome to the CICE Consortium ! First, thanks a lot for considering to contribute to CICE.
+
+There are many ways to contribute, and we appreciate your efforts. For an introductory tour, head over to our [Contribution guide][contributing] and take a look at the [Resources page], our go-to index with links to code, documentation, testing information, guidelines, and more.
+
+[contributing]: https://github.com/CICE-Consortium/About-Us/wiki/Contributing
+[resources]: https://github.com/CICE-Consortium/About-Us/wiki/Resource-Index
+
+## Getting help
+If you still need help getting started using the model afer reviewing the [model documentation][doc-running], the first point of contact with the CICE Consortium is the [Consortium Community Forum][forum]. 
```

Comment:
My suggestion would be to link the documentation to the Resources Page "Model Documentation" section rather than directly into RTD. Sometimes RTD does weird, unexpected things so I think being one step removed would be preferable.

https://github.com/CICE-Consortium/About-Us/wiki/Resource-Index#model-documentation

Reply:
There is a type here "afer".